### PR TITLE
Add aggregated role

### PR DESCRIPTION
**Pull request by itdove**
Signed-off-by: Dominique Vernier <dvernier@redhat.com>

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ spec:
               fieldPath: metadata.namespace          
         - name: DEX_OPERATOR_IMAGE
           value: "quay.io/identitatem/dex-operator:latest"
-        image: quay.io/identitatem/idp-mgmt-operator@sha256:f13d0e986746488a5033e16a4dc0e5a0be8e2f3024f72b680cf8709308fd6fd9
+        image: quay.io/identitatem/idp-mgmt-operator@sha256:a6ab15bc688acb1894d965f6b3f5ca68d944a77d40958a1e64a771fd784f1177
```
Comment (chrisahl) [inline config/manager/manager.yaml:37]: Should this be `latest`?  Should we add a .gitignore rule for this file?

**Review by chrisahl — CHANGES_REQUESTED** [review]
See inline comment

File: config/manager/manager.yaml (hunk 1 above)
Comment (itdove) [reply config/manager/manager.yaml:37]: yes it should, I will change it

**Review by chrisahl — APPROVED** [review]

**Pull request merged**